package.json

```diff
@@ -118,7 +118,7 @@
       "got@14.4.5": "patches/got@14.4.5.patch",
       "slonik@30.4.4": "patches/slonik@30.4.4.patch",
       "@oclif/core@3.26.6": "patches/@oclif__core@3.26.6.patch",
-      "oclif@4.13.6": "patches/oclif@4.13.6.patch",
+      "oclif": "patches/oclif.patch",
```

Comment:
ah cool, I didn't know we could point to a patch without a version. it can solve some of the failures in ci with Renovate